<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigBasicFrame</class>
 <widget class="QFrame" name="ConfigBasicFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QPushButton" name="bModPwd">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>555</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bModPwd
{
background-image: url(:/images/cfgModPwd_Bt.png);
}
#bModPwd:pressed
{
background-image: url(:/images/cfgModPwdDn_Bt.png);
}
#bModPwd:focus{outline: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bCancelBut">
   <property name="geometry">
    <rect>
     <x>915</x>
     <y>555</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bCancelBut
{
background-image: url(:/images/cfgCancel_Bt.png);
}
#bCancelBut:pressed
{
background-image: url(:/images/cfgCancelDn_Bt.png);
}
#bCancelBut:focus{outline: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bExitPwd">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>555</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bExitPwd
{
background-image: url(:/images/cfgExitPwd_Bt.png);
}
#bExitPwd:pressed
{
background-image: url(:/images/cfgExitPwdDn_Bt.png);
}
#bExitPwd:focus{outline: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="basicWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#basicWidget
{
background-image: url(:/images/cfgBasic_Bg.png);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bOkBut">
   <property name="geometry">
    <rect>
     <x>737</x>
     <y>555</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bOkBut
{
background-image: url(:/images/cfgOkBut_Bt.png);
}
#bOkBut:pressed
{
background-image: url(:/images/cfgOkButDn_Bt.png);
}
#bOkBut:focus{outline: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bEditBut">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>555</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bEditBut
{
background-image: url(:/images/cfgEditBut_Bt.png);
}
#bEditBut:pressed
{
background-image: url(:/images/cfgEditButDn_Bt.png);
}
#bEditBut:focus{outline: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
